test/javascripts/unit/components/query-result-test.js.es6

```diff
@@ -0,0 +1,103 @@
+import { moduleFor } from "ember-qunit";
+import { test } from "qunit";
+
+moduleFor("component:query-result");
+
```

Comment:
I see that there are just a few unit tests for components in Discourse. But we need to test data for the chart somehow. Chart renders on canvas, so we can't check the rendered result. The solution is to check computed properties. As I know, it's impossible to do it in integration tests. That's why I'm using unit tests here.